sources/packages/api/src/services/application/application.service.ts

```diff
@@ -864,7 +864,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
         overwrittenStatus: ApplicationStatus.overwritten,
       })
       .orderBy(
-        `CASE application.pirStatus
+        `CASE application.pir_status
```

Comment:
Please add a TODO for further investigation on why `application.pirStatus` was not translated to the DB column name. This will be taken care is the next PR, right?